benchmarking/run_all_benchmarks.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+python -m imitation.scripts.train_imitation bc with bc_seals_ant seed=1
```

Comment:
I know that this can be easily be generated. I used:
```python
envs = [
    "seals_ant",
    "seals_half_cheetah",
    "seals_hopper",
    "seals_swimmer",
    "seals_walker",
]

for env in envs:
    for algo in ["bc", "dagger"]:
        for seed in range(1,11):
            print(f"python -m imitation.scripts.train_imitation {algo} with {algo}_{env} seed={seed}")

    for algo in ["airl", "gail"]:
        for seed in range(1, 11):
            print(f"python -m imitation.scripts.train_adversarial {algo} with {algo}_{env} seed={seed}")
```

However, I opted for solution with less moving parts. Whenever we add new algorithms or change something else I can just re-write this script in 3min. As soon as we put the code in the repo we have to maintain it, test it, document it. I think that is a lot of effort just to safe us those 3mins to re-write this on-off script.

Reply:
I suppose that's true.

Having the `run_all_benchmarks.sh` file in the repo feels slightly weird. Do you think we can remove the file entirely and just keep the above python/bash script that generates those commands? There doesn't seem any need to have the `run_all_benchmarks.sh` script at all.

Reply:
I don't think this matters much either way, but I have a slight preference for the current approach because I think it makes it easier to see exactly what was executed, and also to run just a subset of the experiments by copy-pasting.